### ISSUE TITLE
[MSKINS-24] Add the Twitter follow button support

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-fluido-skin#135]
**[Simone Tripodi](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=simone.tripodi)** opened **[MSKINS-24](https://issues.apache.org/jira/browse/MSKINS-24?redirect=false)** and commented

With few configuration parameters in the `custom.fluidoSkin` section in `site.xml` it is possible to add [Twitter follow button](https://dev.twitter.com/docs/follow-button) support

---

**Affects:** fluido-1.1

**Comment by jira-importer** [comment]
**[Simone Tripodi](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=simone.tripodi)** commented

Fixed on [r1227319](https://svn.apache.org/viewvc?view=revision&revision=1227319)

**Comment by jira-importer** [comment]
**[Lars Francke](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=lars_francke)** commented

Looks like there's a "http:" missing.

It only produces

```
js.src = "//platform.twitter.com/widgets.js";
```

instead of

```
js.src = "http://platform.twitter.com/widgets.js";
```

**Comment by jira-importer** [comment]
**[Simone Tripodi](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=simone.tripodi)** commented

did you try it? It is currently working, see the related [IT](http://maven.apache.org/skins/maven-fluido-skin/mskins-24_topbar/index.html)

Have a look at the Twitter's [Follow Button](https://dev.twitter.com/docs/follow-button) documentation, read the section _Using Javascript_ to see why it is missing ;)

**Comment by jira-importer** [comment]
**[Lars Francke](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=lars_francke)** commented

I tried, yeah and it does not work locally. I now tried to actually deploy a site and then the JS seems to be working. The _Using Javascript_ section actually has multiple different snippets and some actually do include the `http:` scheme. Don't know why it's missing in some of them and is there in others.

Sorry for the confusion but I didn't want to deploy the site with the button not working...but turns out I should have just done it ;-)
